### ISSUE TITLE
About data keep ratio

### ISSUE THREAD

**Issue by MatsumotoNana78** [magic-research/Dataset_Quantization#12]
Thank you for your wonderful paper!
I have one question:
Are DataKeepRatio and PatchDropRatio related?
In the general coreset methods, for example, if the full dataset size is 50000 and the coreset size is 5000, then DataKeepRatio=0.1(5000/50000). 
Is the Dataset Quantization the same?
In Figure 4.(d) of the paper, DataKeepRatio appears to be correlated with PatchDropRatio. i.e. Why are the x-coordinates of the data points not aligned?
![image](https://github.com/user-attachments/assets/97f1e2fa-f696-4176-beaf-22ca51ee72fa)

**Comment by vimar-gu** [comment]
Thanks for the question.

Yes, the final data keep ratio is sample_keep_ratio * patch_keep_ratio. In the experiments of this paper, we drop 20% patches for each image, so the data points in the figure are moved to left.

**Comment by mmnn-00** [comment]
Thank you for your reply^^.
By the way, is "Data keep ratio" on the horizontal axis in figure5 also sample_keep_ratio * patch_keep_ratio?
![image](https://github.com/user-attachments/assets/5129f92d-9c33-4874-872d-900a909f5cec)

**Comment by vimar-gu** [comment]
Yes. In this figure, in order that we can compare with previous method under the same data keep ratio, the original sample keep ratio is multiplied with 1.25, so that after dropping patches, the final data keep ratio is the same as other methods.

**Comment by mmnn-00** [comment]
Thank you very much!!
